indico/modules/events/contributions/controllers/management.py

```diff
@@ -456,17 +466,60 @@ def _process(self):
         return send_file('contributions.pdf', pdf.generate(), 'application/pdf')
 
 
-class RHContributionsExportPDFBook(RHManageContributionsExportActionsBase):
+class RHContributionExportTexConfig(RHManageContributionsExportActionsBase):
+    ALLOW_LOCKED = True
     def _process(self):
-        pdf = ContributionBook(self.event, session.user, self.contribs, tz=self.event.timezone)
-        return send_file('book-of-abstracts.pdf', pdf.generate(), 'application/pdf')
+        form = ContributionExportTeXForm(contribs=self.contribs)
 
+        if form.validate_on_submit():
+            data = form.data
+            data.pop('submitted', None)
+            data['contribution_ids'] = [x.id for x in self.contribs]
+            key = unicode(uuid.uuid4())
+            export_list_cache.set(key, data, time=1800)
+            download_url = url_for('.contributions_tex_export_book', self.event, uuid=key)
+            return jsonify_data(flash=False, redirect=download_url, redirect_no_loading=True)
 
-class RHContributionsExportPDFBookSorted(RHManageContributionsExportActionsBase):
-    def _process(self):
-        pdf = ContributionBook(self.event, session.user, self.contribs, tz=self.event.timezone,
-                               sort_by='board_number')
-        return send_file('book-of-abstracts.pdf', pdf.generate(), 'application/pdf')
+        return jsonify_template('events/contributions/management/exportdialog.html', form=form, event=self.event)
+
+
+class RHContributionsExportTeXBook(RHManageContributionsExportActionsBase):
+    ALLOW_LOCKED = True
+    normalize_url_spec = {
```

Comment:
I don't think there is any URL normalization defined in the parent classes, so this is not necessary. Especially since a normalization spec without any locators is useless, since it won't do anything.